packages/playground/website/playwright/e2e/website-ui.spec.ts

```diff
@@ -0,0 +1,175 @@
+import { test, expect } from '../playground-fixtures.ts';
+import {
+	clickCreateInNewSiteModal,
+	clickSaveInEditSettings,
+	expandSiteView,
+	getSiteInfoRowValue,
+	getSiteTitle,
+	hasNetworkingEnabled,
+	openEditSettings,
+	openNewSiteModal,
+	selectPHPVersion,
+	selectWordPressVersion,
+	setNetworkingEnabled,
+} from './website-helpers.ts';
+import { Blueprint } from '@wp-playground/blueprints';
+
+// We can't import the SupportedPHPVersions versions directly from the remote package
+// because of ESModules vs CommonJS incompatibilities. Let's just import the
+// JSON file directly. @ts-ignore
+// eslint-disable-next-line @nx/enforce-module-boundaries
+import { SupportedPHPVersions } from '../../../../php-wasm/universal/src/lib/supported-php-versions.ts';
+// eslint-disable-next-line @nx/enforce-module-boundaries
+import * as MinifiedWordPressVersions from '../../../wordpress-builds/src/wordpress/wp-versions.json';
+
+test('should reflect the URL update from the navigation bar in the WordPress site', async ({
+	page,
+}) => {
+	await page.goto('./?url=/wp-admin');
+
+	await expandSiteView(page);
+
+	const inputElement = page.locator('input[value="/wp-admin/"]');
+	await expect(inputElement).toBeVisible();
+	await expect(inputElement).toHaveValue('/wp-admin/');
+});
+
+test('should switch between sites', async ({ page }) => {
+	await page.goto('./');
+
+	await openNewSiteModal(page);
+
+	const newSiteName = await page
+		.locator('input[placeholder="Playground name"]')
+		.inputValue();
+
+	await clickCreateInNewSiteModal(page);
+
+	await expect(await getSiteTitle(page)).toMatch(newSiteName);
+
+	const sidebarItem = page.locator(
+		'.components-button[class*="_sidebar-item"]:not([class*="_sidebar-item--selected_"])'
+	);
+	const siteName = await sidebarItem
+		.locator('.components-flex-item[class*="_sidebar-item--site-name"]')
+		.innerText();
+	await sidebarItem.isVisible();
+	await sidebarItem.click();
+
+	await expect(await getSiteTitle(page)).toMatch(siteName);
+});
+
+SupportedPHPVersions.forEach(async (version) => {
+	test(`should switch PHP version to ${version}`, async ({ page }) => {
+		/**
+		 * WordPress 6.6 dropped support for PHP 7.0 and 7.1 so we need to skip these versions.
+		 * @see https://make.wordpress.org/core/2024/04/08/dropping-support-for-php-7-1/
+		 */
+		if (['7.0', '7.1'].includes(version)) {
+			return;
+		}
+		await page.goto(`./`);
+
+		await openEditSettings(page);
+
+		await selectPHPVersion(page, version);
+
+		await clickSaveInEditSettings(page);
+
+		expect(await getSiteInfoRowValue(page, 'php-version')).toMatch(
+			`${version} (with extensions)`
+		);
```

Comment:
This approach can lead to flaky results as we're not waiting for the values to match. Since the `getSiteInfoRowValue` uses locator internally, I think we should just return that locator and assert against it, for example:

```js
// the utility
function getSiteInfoLocator() {
  return page.locator( '#site-info-row' );
}

// the assertion
await expect(getSiteInfoLocator()).toHaveText(`${version} (with extensions)`);
```